pkg/daemon/multus/resources.go

```diff
@@ -349,16 +370,13 @@ func (vt *ValidationTest) numClientsReady(ctx context.Context, expectedNumPods i
 	return numReady, nil
 }
 
-func (vt *ValidationTest) getClientPods(ctx context.Context, expectedNumPods int) (*core.PodList, error) {
+func (vt *ValidationTest) getPodsWithLabel(ctx context.Context, label string) (*core.PodList, error) {
```

Comment:
There is an existing method `k8sutil.PodsRunningWithLabel()`. Would it make sense to call that method instead? Or maybe move this helper into `k8sutil` if it's different?

Reply:
`k8sutil.PodsRunningWithLabel()` only lists pods in Running state. This method needs to list pods regardless of state.